src/storage/src/hummock/compactor/compactor_runner.rs

```diff
@@ -142,6 +142,11 @@ impl CompactorRunner {
         task_progress: Arc<TaskProgress>,
     ) -> HummockResult<impl HummockIterator<Direction = Forward>> {
         let mut table_iters = Vec::new();
+        let compact_io_retry_time = self
+            .compactor
+            .context
+            .storage_opts
+            .object_store_streaming_read_timeout_ms;
```

Comment:
object_store_streaming_read_timeout_ms is already used by a more fine-grained operation. I suggest another new option.

Reply:
+1